libs/settle/src/Comparator/InArray.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Settle\Comparator;
+
+/**
+ * @template TValue
+ * @implements ComparatorInterface<TValue>
+ */
+class InArray implements ComparatorInterface
+{
+    private array $targetValues;
+    private bool $strict;
+
+    /**
+     * @phpstan-param TValue[] $targetValues
+     * @param mixed $targetValues
+     */
+    public function __construct(array $targetValues, bool $strict)
```

Comment:
Mozna bych tomu $strict dal default value `false`. Nebo to ma nejaky duvod proc by to nemelo mit default?

Reply:
Aha, no to bude asi teda ten důvod proč tam není default :D

Reply:
No nevím, nechám to radši bejt